examples/join-multiple-streams/join_multiple_streams.bal

```diff
@@ -62,16 +62,14 @@ function printMaterialUsageAlert(MaterialUsage materialUsage) {
 
 listener http:Listener productMaterialListener = new (9090);
 
+future<()> ftr = start initRealtimeProductionAlert();
```

Comment:
This can be done with __init() method now.

Reply:
fixed